binding/angular/package.json

```diff
@@ -33,9 +32,10 @@
     "@angular-eslint/template-parser": "1.2.0",
     "@angular/cli": "~11.0.5",
     "@angular/compiler-cli": "~11.0.5",
-    "@picovoice/web-voice-processor": "^2.2.2",
+    "@picovoice/porcupine-web": "=2.1.7",
+    "@picovoice/web-voice-processor": "~4.0.0",
     "@types/jasmine": "~3.6.0",
-    "@types/node": "^12.11.1",
+    "@types/node": "^14.0.0",
```

Comment:
both the binding project and the demo project have a bunch of bloat that come from `karma` and `jasmine`, which are testing libraries that we don't use. Can you remove these from the `package.json` and the projects?